### ISSUE TITLE
remove getindex.jl

### ISSUE THREAD

**Issue by milktrader** [JuliaQuant/FinancialSeries.jl#3]
Doubling the code from TimeSeries just doubles the maintenance and serves no useful purpose.

**Comment by multidis** [comment]
Yes I commented it out already in `FinancialSeries.jl` source file, but had not checked yet if any of its code is needed for `TickData`.

**Comment by milktrader** [comment]
Good point! I suppose that we then just need to remove references to FinancialTimeSeries and let that file still live to support the TickData type for now.
